releasenotes/notes/change-set-external-tags-api-e5d6f80ad7c8376f.yaml

```diff
@@ -0,0 +1,16 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    A change in the 'set_external_tags' function from the 'datadog_agent' Python
```

Comment:
We still drop invalid tags silently but with #3562 we handle "ascii unicode" like before. I think we should still keep this note as all direct call to Go functions from Python should go through the base class.